seed/challenges/01-front-end-development-certification/intermediate-bonfires.json

```diff
@@ -730,21 +730,21 @@
         "Remember to use <a href='//github.com/FreeCodeCamp/freecodecamp/wiki/How-to-get-help-when-you-get-stuck' target='_blank'>Read-Search-Ask</a> if you get stuck. Try to pair program. Write your own code."
       ],
       "challengeSeed": [
-        "function steamroller(arr) {",
+        "function steamrollArray(arr) {",
```

Comment:
@SaintPeter this challenge (Steamroller) wasn't on your list. I assume that it's because you thought the function name was fairly unique enough already. So unnecessary change in this PR?